packages/fxa-auth-server/lib/senders/email.js

```diff
@@ -12,6 +12,7 @@ const url = require('url');
 const i18n = require('i18n-abide');
 const { URL } = url;
 const { productDetailsFromPlan } = require('fxa-shared').subscriptions.metadata;
+const LocalizeEmails = require('./emails/localizeEmails');
```

Comment:
I'll leave a few comments wherever necessary to throw some light on the changes relevant to this issue